tests/payload_tools/auth.py

```
@@ -1,15 +1,17 @@
 
-def build_signup_payload(email_prefix="testuser", password="StrongPass123", name="Test User", age=30, gender="non-binary"):
-    """Builds a mock user signup payload."""
-    timestamp = dt.datetime.now().timestamp()
-    return {
-        "email": f"{email_prefix}_{timestamp}@example.com",
-        "password": password,
-        "name": name,
-        "age": age,
-        "gender": gender,
-    }
+def build_signup_payload(email_prefix="testuser", password="StrongPass123", name="Test User", age=30, gender="non-binary", pfp_url=None, bio=None):
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to function default: "password"

(S107)